qualtran/surface_code/beverland_et_al_model.py

```diff
@@ -116,14 +116,12 @@ def n_discrete_logical_gates(
         rotation_model: Cost model used to compute the number of T gates
             needed to approximate rotations.
     """
-    n_rotations: int = alg.n_logical_gates.rotation
-    ret = attrs.evolve(alg.n_logical_gates, rotation=0)
-    if n_rotations > 0:
-        ret = (
-            ret
-            + rotation_model.preparation_overhead(eps_syn)
-            + n_rotations * rotation_model.rotation_cost(eps_syn / n_rotations)
-        )
+    rotation_epsilons: dict[float, int] = alg.n_logical_gates.rotation_epsilons
+    ret = attrs.evolve(alg.n_logical_gates, rotation_epsilons={})
+    if rotation_epsilons:
+        rotation_model.preparation_overhead(min(eps for eps in rotation_epsilons.values()))
```

Comment:
what's the correct way to pick the eps for this?

Reply:
`eps_syn` is passed in to this function

Reply:
fixed to ignore bloq eps, as discussed